src/Kiota.Builder/WorkspaceManagement/WorkspaceManagementService.cs

```diff
@@ -156,7 +156,7 @@ public async Task<bool> ShouldGenerateAsync(GenerationConfiguration inputConfig,
     {
         if (!UseKiotaConfig || cleanOutput)
             return null;
-        return await descriptionStorageService.GetDescriptionAsync(clientName, new Uri(inputPath).GetFileExtension(), cancellationToken).ConfigureAwait(false);
+        return await descriptionStorageService.GetDescriptionAsync(clientName, Path.GetExtension(inputPath), cancellationToken).ConfigureAwait(false);
```

Comment:
what's the rationale behind that change?

Reply:
File paths are not valid uris. This allows using local files.

Reply:
the argument name might be misleading but as far as I remember this section of the code, both a URI and a local path can be provided here.
If parsing a file path is an issue, having a reusable method that can accept both a path or a uri and give accurate results would be better since this extension method is used multiple places.

Reply:
Reverted. Issue is that we were not normalizing paths provided during edit command.